test/setup.js

```diff
@@ -6,16 +6,18 @@
  * https://github.com/airbnb/enzyme/blob/master/docs/api/mount.md
  */
 
-require('jsdom-global')()
-require('./polyfills/MutationObserver.js')(global)
-require('./polyfills/getSelection.js')(global)
+require('jsdom-global')();
+require('./polyfills/MutationObserver.js')(global);
+require('./polyfills/getSelection.js')(global);
 
 // Configure the Enzyme adapter
 const Enzyme = require('enzyme');
 const EnzymeAdapter = require('enzyme-adapter-react-16');
-Enzyme.configure({adapter: new EnzymeAdapter()});
+Enzyme.configure({ adapter: new EnzymeAdapter() });
 
 // Setup Chai to use Enzyme
 const chai = require('chai');
 const chaiEnzyme = require('chai-enzyme');
 chai.use(chaiEnzyme());
+
+global.expect = chai.expect;
```

Comment:
Moved this here instead of in the test file so that the tests use the chai provided by Cypress when running there